sql/item_mods.sql

```diff
@@ -44161,11 +44161,11 @@ INSERT INTO `item_mods` VALUES (21756,506,130); -- EXTRA_DMG_CHANCE: 130
 INSERT INTO `item_mods` VALUES (21756,507,200); -- OCC_DO_EXTRA_DMG: 200
 INSERT INTO `item_mods` VALUES (21756,659,40);  -- WSD+% (Metatron Torment): 40
 
--- Conqueror (119 III)
```

Comment:
I think we should retain the previous verion of the item name comments so that we can tell apart _which_ version of Conqueror it is at a glance

Reply:
Nice catch! I had undone the mod rename and didn't get those back in there.

I have added the level comments back in the same format that BG Wiki has them here:
https://www.bg-wiki.com/ffxi/Conqueror